eng/.docsettings.yml

```diff
@@ -18,4 +18,28 @@ known_presence_issues:
   - ['sdk/keyvault','#5499']
 known_content_issues:
   - ['README.md','Root readme']
-  - ['sdk/keyvault/Microsoft.Azure.KeyVault/README.md','#5499']
\ No newline at end of file
+  - ['sdk/keyvault/Microsoft.Azure.KeyVault/README.md','#5499']
+# .net climbs upwards. placing these to prevent assigning readmes to the wrong project
+package_indexing_exclusion_list:
```

Comment:
These are the discovered csprojs that I have judged to NOT be correlated to an upstream package. If anything needs update here, please sing out.